account_asset_management_xls/report/account_asset_report_xls.py

```diff
@@ -648,7 +654,7 @@ def _active_report(self, _p, _xs, data, objects, wb):
         # nova start: Get all assets between dates
         cr.execute(
             "SELECT id FROM account_asset "
-            "WHERE date_start <= %s"
+            "WHERE purchase_date <= %s "
```

Comment:
There are assets in the system with date_start < purchase_date. Should we perhaps check both dates?

Reply:
Thanks, missed that part.